src/Tablebot/Plugins/Netrunner.hs

```diff
@@ -0,0 +1,269 @@
+
+*Usage:*
+  - `netrunner find card name` -> fetches the image of the card matching "card name"
+  - `{{!card name}}          ` -> the inline version of the above command|]
+    []
+    None
+
+findFlavourHelp :: HelpPage
+findFlavourHelp =
+  HelpPage
+    "flavour"
+    "searches the NetrunnerDB database for a card's flavour text"
+    [r|**Find Netrunner Card Flavour Text**
+Searches the NetrunnerDB database for the card closest matching a given query and shows its flavour text
+Can be used inline by enclosing your query inside curly braces with a `|` (max five queries per message)
+
+*Usage:*
+  - `netrunner find card name` -> fetches the flavour text of the card matching "card name"
+  - `{{|card name}}          ` -> the inline version of the above command|]
+    []
+    None
+
+customHelp :: HelpPage
+customHelp =
+  HelpPage
+    "custom"
+    "generates custom Netrunner cards"
+    [r|**Create Custom Netrunner Cards**
+Generates custom Netrunner cards and formats them like existing cards
+The order of card parameters does not matter
+If you mispell a card parameter (e.g. "typ" instead of "type") it will attempt to correct it
+
+*Usage:*
+- `netrunner custom type:agenda                ` -> creates an agenda
+- `netrunner custom title:"Name" text:"Lorem." ` -> creates a card with a title and text
+- `netrunner custom faction:"nbn"              ` -> creates a card with a faction
+- `netrunner custom keywords:"AP - Hardware"   ` -> creates a card with subtypes
+- `netrunner custom advancement:5 points:3     ` -> creates a card with an advancement requirement and agenda points
+- `netrunner custom cost:3 trash:2             ` -> creates a card with play/rez cost and trash cost
+- `netrunner custom strength:4                 ` -> creates a card with strength
+- `netrunner custom minSize:40 maxInf:15 link:2` -> creates a card with a minimum deck size, maximum influence, and link
+- `netrunner custom flavour:"Raspberry & mint" ` -> creates a card with flavour text
+- `netrunner custom unique:true                ` -> creates a unique card|]
+    []
+    None
+
+beginnerText :: EnvDatabaseDiscord NrApi Text
+beginnerText = do
+  subroutine <- formatFromEmojiName "subroutine"
+  agenda <- formatFromEmojiName "agenda"
+  rezCost <- formatFromEmojiName "rez_cost"
+  return $
+    agenda <> " **NETRUNNER** " <> rezCost
+      <> [r|
+Netrunner is an asymmetric collectable card game about hackers hacking corporations. It's run as a *free* community endeavour by NISEI:
+|]
+      <> subroutine
+      <> [r| <https://nisei.net/>
+
+**Learn to play**
+|]
+      <> subroutine
+      <> [r| <https://nisei.net/players/learn-to-play/>
+
+**Get involved here**
+There is a sizeable Netrunner community here of new and old society members. If you want to get into the game feel free to ask in #netrunner for some advice or a beginner game and someone will be happy to help you!|]
+
+netrunnerStartUp :: StartUp NrApi
+netrunnerStartUp = StartUp $ liftIO getNrApi
```

Comment:
We might want this to be periodically refreshed? An MVar and a cron job would probably be the easiest way to do that. Beanstalk has a command that does it, but I suppose !reload could be used in that way already

Reply:
`.reload` will suffice for now, only because the database of Netrunner cards is updated very slowly (new releases are once every half a year if we're lucky, volunteer organisations are slow)

Reply:
Fair. I'm used to mtg (from when I started the tabletop deck feature that I never finished), where daily updating is recommended.

Reply:
all good!